styles.css

```diff
@@ -195,6 +195,37 @@ html {
   }
 }
 
+/* skills */
+
+.skills-panel .container {
+  margin: 0 auto;
+  width: 70%;
+  border-top: 2px solid #c4c4c4;
+  margin-top: 30px;
```

Comment:
Combine the margin-top with the margin: 0 auto; on line 201. When setting more than one margin-top type rule you should always combine them into a single margin: 0 0 0 0; rule.